### DISCUSSION TITLE
Contact sur le projet Pi

### DISCUSSION THREAD

**Discussion by francoistonic** [pcouy/rpi-eth-display#1] [General]
Bonjour,

Bravo pour le développement Pi et le port vidéo. Cela m'intéresse beaucoup pour en parler dans le magazine Programmez! 

Peut-on échanger sur le sujet ? ftonic@programmez.com

**Comment by pcouy** [comment]
Je viens de vous envoyer un mail (je précise ici pour que vous puissiez vérifier avec le timestamp que mon identité n'est pas usurpée)
